token-metadata/program/src/processor/mod.rs

```diff
@@ -201,39 +201,39 @@ fn process_legacy_instruction<'a>(
             )
         }
         MetadataInstruction::DeprecatedCreateMasterEdition(_args) => {
-            msg!("Instruction: Deprecated Create Master Edition, Removed in 1.1.0");
+            msg!("IX: Deprecated Create Master Edition, Removed in 1.1.0");
             Err(MetadataError::Removed.into())
         }
         MetadataInstruction::DeprecatedMintNewEditionFromMasterEditionViaPrintingToken => {
-            msg!("Instruction: Deprecated Mint New Edition from Master Edition Via Token, Removed in 1.1.0");
+            msg!("IX: Deprecated Mint New Edition from Master Edition Via Token, Removed in 1.1.0");
```

Comment:
For the ones that have been removed, perhaps we can remove the whole message – not necessarily now.

Reply:
Yeah that's a good call. I didn't notice how verbose those messages are until today.

Reply:
Good point